extensions-core/orc-extensions/src/main/java/org/apache/druid/data/input/orc/OrcInputFormat.java

```diff
@@ -60,6 +60,8 @@ public boolean isSplittable()
   @Override
   public InputEntityReader createReader(InputRowSchema inputRowSchema, InputEntity source, File temporaryDirectory)
   {
+    OrcExtensionsModule orcExtensionsModule = new OrcExtensionsModule();
```

Comment:
we should not create a GUICE module here. why can't we move the `initialize` function here itself?